src/status_im/contexts/preview/quo/slideshow/slider_bar.cljs

```diff
@@ -0,0 +1,49 @@
+(ns status-im.contexts.preview.quo.slideshow.slider-bar
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.contexts.preview.quo.preview :as preview]))
+
+(def descriptor
+  [{:key  :total-amount
+    :type :number}
+   {:label "Blur (dark only)?"
+    :key   :blur?
+    :type  :boolean}
+   (preview/customization-color-option)])
+
+(defn view
+  []
+  (let [[state set-state] (rn/use-state {:total-amount        10
```

Comment:
I believe there is an updated way to get state.
cc @ajayesivan perhaps you can point to this? 🤔